flask_restplus/templates/swagger-ui.html

```diff
@@ -1,78 +1,91 @@
         </symbol>
 
         <symbol viewBox="0 0 20 20" id="large-arrow">
-          <path d="M13.25 10L6.109 2.58c-.268-.27-.268-.707 0-.979.268-.27.701-.27.969 0l7.83 7.908c.268.271.268.709 0 .979l-7.83 7.908c-.268.271-.701.27-.969 0-.268-.269-.268-.707 0-.979L13.25 10z"/>
+            <path d="M13.25 10L6.109 2.58c-.268-.27-.268-.707 0-.979.268-.27.701-.27.969 0l7.83 7.908c.268.271.268.709 0 .979l-7.83 7.908c-.268.271-.701.27-.969 0-.268-.269-.268-.707 0-.979L13.25 10z"/>
         </symbol>
 
         <symbol viewBox="0 0 20 20" id="large-arrow-down">
-          <path d="M17.418 6.109c.272-.268.709-.268.979 0s.271.701 0 .969l-7.908 7.83c-.27.268-.707.268-.979 0l-7.908-7.83c-.27-.268-.27-.701 0-.969.271-.268.709-.268.979 0L10 13.25l7.418-7.141z"/>
+            <path d="M17.418 6.109c.272-.268.709-.268.979 0s.271.701 0 .969l-7.908 7.83c-.27.268-.707.268-.979 0l-7.908-7.83c-.27-.268-.27-.701 0-.969.271-.268.709-.268.979 0L10 13.25l7.418-7.141z"/>
         </symbol>
 
 
         <symbol viewBox="0 0 24 24" id="jump-to">
-          <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"/>
+            <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"/>
         </symbol>
 
         <symbol viewBox="0 0 24 24" id="expand">
-          <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/>
+            <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/>
         </symbol>
 
-      </defs>
-    </svg>
-
-    <div id="swagger-ui"></div>
-
-    {% include 'swagger-ui-libs.html' %}
-    <script type="text/javascript">
-        window.onload = function() {
-            const ui = window.ui = new SwaggerUIBundle({
-                url: "{{ specs_url }}",
-                validatorUrl: "{{ config.SWAGGER_VALIDATOR_URL }}" || null,
-                dom_id: "#swagger-ui",
-                presets: [
-                    SwaggerUIBundle.presets.apis,
-                    SwaggerUIStandalonePreset.slice(1) // No Topbar
-                ],
-                plugins: [
-                    SwaggerUIBundle.plugins.DownloadUrl
-                ],
-                displayOperationId: {{ config.SWAGGER_UI_OPERATION_ID|default(False)|tojson }},
-                displayRequestDuration: {{ config.SWAGGER_UI_REQUEST_DURATION|default(False)|tojson }},
-                docExpansion: "{{ config.SWAGGER_UI_DOC_EXPANSION | default('none') }}"
-            })
-
-            {% if config.SWAGGER_UI_OAUTH_CLIENT_ID -%}
+    </defs>
+</svg>
+
+<div id="swagger-ui"></div>
+
+{% include 'swagger-ui-libs.html' %}
+<script>
+    window.onload = function () {
+
+        // Build a system
+        const ui = SwaggerUIBundle({
+            url: "http://petstore.swagger.io/v2/swagger.json",
```

Comment:
This should be removed, as it's overwritten two lines below

Reply:
You're right. Removed, thanks for the tip!